### DISCUSSION TITLE
2.0.2

### DISCUSSION THREAD

**Discussion by LuqueDaniel** [renpy/vscode-language-renpy#43] [Releases]
## What's Changed
* Improve folding function and grammar. by @joe69-96 in https://github.com/LuqueDaniel/vscode-language-renpy/pull/32
* Add settings for diagnostic warnings by @rdurfee in https://github.com/LuqueDaniel/vscode-language-renpy/pull/35
  * Diagnostic warnings can now be disabled for reserved variable names, invalid variable names, or invalid filenames.
  * Reserved Ren'Py variables such as `_window_subtitle` no longer flag an error or warning
  * Variable names with numbers now highlight properly in Python functions
* Defined variables and persistents are now recognized without a compile by @rdurfee in https://github.com/LuqueDaniel/vscode-language-renpy/pull/37
  * Adding default/define variables and persistents are now recognized without a compile [#27](https://github.com/LuqueDaniel/vscode-language-renpy/issues/27)
  * Auto-complete now functions correctly for internal classes such as `iap` and `updater` [#36](https://github.com/LuqueDaniel/vscode-language-renpy/issues/36)
  * Documentation for reserved Ren'Py variables such as `_window_subtitle` now displays correctly [#36](https://github.com/LuqueDaniel/vscode-language-renpy/issues/36)
* Prepare 2.0.2 release by @LuqueDaniel in https://github.com/LuqueDaniel/vscode-language-renpy/pull/42

**Full Changelog**: https://github.com/LuqueDaniel/vscode-language-renpy/compare/v2.0.1...v2.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/LuqueDaniel/vscode-language-renpy/releases/tag/v2.0.2'>2.0.2</a>.</em>

**Comment by qirien** [comment]
Thank you for all the recent improvements! I especially love the color picker and function documentation.
